kata-deploy/action/Dockerfile

```diff
@@ -0,0 +1,31 @@
+# Copyright (c) 2018 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+FROM microsoft/azure-cli:latest
+
+LABEL com.github.actions.name="Test kata-deploy in an AKS cluster"
+LABEL com.github.actions.description="Test kata-deploy in an AKS cluster"
+
+# Default to latest validated AKS-engine version
+ARG AKS_ENGINE_VER="v0.42.0"
+ARG ARCH=amd64
+
+ENV GITHUB_ACTION_NAME="Test kata-deploy in an AKS cluster"
+
+# When run, we expect the caller (GitHub Action workflow) to provide the
+# PKG_SHA environment variable
+ENV PKG_SHA=HEAD
+
+RUN curl -LO https://storage.googleapis.com/kubernetes-release/release/$(curl -s https://storage.googleapis.com/kubernetes-release/release/stable.txt)/bin/linux/${ARCH}/kubectl \
```

Comment:
I'm not sure if this line is pure evil or pure genius :angel: :smiling_imp: :smile:

Reply:
I like it :) . I would prefer to have this option for all the components (see aks-engine above)